docs/src/modules/components/DiamondSponsors.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
```

Comment:
I think I see this fairly often. Please annotate specific lines so that it's obvious why this is required. 

This rule very rarely flags false positive missing translations.

Reply:
Ah right, thanks, we can remove it. I think that it comes from a copy and paste. I didn't notice it.